resources/lib/tvguide.py

```diff
@@ -203,6 +212,19 @@ def get_episode_items(self, date, channel):
             ))
         return episode_items
 
+    @staticmethod
+    def get_episode_path(episode, channel, noop=False):
+        """Return a playable plugin:// path for an episode"""
+        now = datetime.now(dateutil.tz.tzlocal())
+        end_date = dateutil.parser.parse(episode.get('endTime'))
+        if episode.get('url') and episode.get('vrt.whatson-id'):
+            return url_for('play_whatson_id', whatson_id=episode.get('vrt.whatson-id'))
+        if now - timedelta(hours=24) <= end_date <= now:
+            return url_for('play_air_date', channel, episode.get('startTime')[:19], episode.get('endTime')[:19])
```

Comment:
I don't think we want this. We are not sure that the play by airdate will work, and we will fallback to the airdate url in IPTV Manager anyway.

When we supply an url, we will also see a green bullet in the EPG, and this should be used to confirm that there is a program available.

Reply:
Okay, I'll remove this.